### DISCUSSION TITLE
Toolbar issues

### DISCUSSION THREAD

**Discussion by thetrapfatheeeeer** [piroor/treestyletab#3379] [Q&A]
How can i hide the "Tree Style Tab" toolbar on the top of the extension?

**Comment by alexUnd** [comment]
if you mean hide the "Tree Style Tab" toolbar on the top of the sidebar, it's described [here](https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#hide-the-tree-style-tab-header-at-the-top-of-the-sidebar)
